impress-org/givewp
# Feature: new form builder settings UI

Resolves [GIVE-21]

## Description

This pull request introduces a new interface for the Donation Form settings in the Visual Form Builder. Instead of the button to switch between "Edit Form" and "Edit Design" modes, there is now a tabs navigation. The settings have been moved from a sidebar in the "Edit Form" view to their own dedicated view. This new view includes a sidebar with a menu on the left side, and the settings are displayed in the main content area. Previously, certain settings were accessed through modals and popovers, but now they are directly available on the page for editing, such as email template options and donation confirmation editors.

It's important to note that no functionality has been changed from the previous version. This work primarily affects the organization of components, so everything continues to function as it did before.

## Affects

Visual Form Builder settings UI only.

## Visuals
https://github.com/impress-org/givewp/assets/3921017/9da16c87-3522-40a5-8957-140a7f8a97cd

## Testing Instructions

1. Open the visual form builder
2. Check that you can edit all settings
3. Check that everything done has the same behavior that if done before this PR

## Pre-review Checklist






[GIVE-21]: https://stellarwp.atlassian.net/browse/GIVE-21?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
